src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs

```diff
@@ -416,6 +416,10 @@ private static void HookupHistory(ModelBuilder modelBuilder)
             .Entity<PackageBlueprintHistory>()
             .HasIndex(e => new { e.PackageEntityId, e.FromStateVersion });
 
+        modelBuilder
+            .Entity<PackageBlueprintHistory>()
+            .HasIndex(e => new { e.Name, e.Version, e.PackageEntityId, e.FromStateVersion });
```

Comment:
Guess we might want to define it as `PackageEntityId, Name, Version, FromStateVersion`.